CMakeLists.txt

```diff
@@ -684,18 +684,19 @@ endif()
 ## Recurse into sources.
 ##
 
-add_subdirectory(client)
-add_subdirectory(combat)
-add_subdirectory(Empire)
-add_subdirectory(network)
-add_subdirectory(parse)
-add_subdirectory(python)
-add_subdirectory(server)
+add_subdirectory(src/common/client)
```

Comment:
can you separate the source layout changes?

Reply:
It's here #3137

Reply:
Removed sources layout changes.